package.nls.json

```diff
@@ -44,6 +45,8 @@
   "configuration.SAS.connectionProfiles.profiles.connectionType": "SAS Profile Connection Type",
   "configuration.SAS.connectionProfiles.profiles.context": "SAS Viya Context",
   "configuration.SAS.connectionProfiles.profiles.endpoint": "SAS Viya Connection Profile Endpoint",
+  "configuration.SAS.connectionProfiles.profiles.fileNavigationCustomRootPath": "File navigation custom root path",
```

Comment:
When doing translation, I realized that it seems we never say "File navigation" in UI. It's a general phrase and may not look obivous when translated. Shall we say something matching the UI (e.g. SAS Server)?

Reply:
Replaced `File navigation` with `SAS Server` in https://github.com/sassoftware/vscode-sas-extension/pull/1557/commits/aef333173872a977a4f148c82e88abf42903d2f2